includes/admin/class-wc-stripe-onboarding-controller.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Admin page for UPE onboarding wizard.
+ *
+ * @since 5.4.1
+ */
+class WC_Stripe_Onboarding_Controller {
+	public function __construct() {
+		add_action( 'admin_menu', [ $this, 'add_onboarding_wizard' ] );
```

Comment:
So that the scripts are not enqueued when not needed
```suggestion
		if ( ! WC_Stripe_Feature_Flags::is_upe_settings_redesign_enabled() ) {
			return;
		}

		add_action( 'admin_menu', [ $this, 'add_onboarding_wizard' ] );
```

Reply:
I am wrapping the feature flag checks in the woocommerce-gateway-stripe level so the file doesn't even need to be loaded.

Reply:
Sounds perfect 👍